Workflow.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Mar 23 12:30:17 2016
+
+@author: mina
+"""
+
+import yaml
+
+class Workflow:
+    def __init__(adress,self):      
+      with open(adress, 'r') as f:
```

Comment:
When reading in the config, we could wrap this section in a try/catch block so that we could provide an error to the user if we were unable to open the workflow config file.